crates/typst/src/geom/angle.rs

```diff
@@ -66,10 +68,32 @@ impl Angle {
         self.to_rad().cos()
     }
 
+    /// Get the sine and cosine of this angle in radians.
+    pub fn sin_cos(self) -> (f64, f64) {
+        self.to_rad().sin_cos()
+    }
+
     /// Get the tangent of this angle in radians.
     pub fn tan(self) -> f64 {
         self.to_rad().tan()
     }
+
+    /// Corrects this angle for the aspect ratio of a gradient.
+    pub fn correct_aspect_ratio(self, aspect_ratio: Ratio) -> Self {
```

Comment:
should this function really live with the `Angle` type? It feels like it belongs with gradients.